flow/scripts/util.tcl

```diff
@@ -40,14 +40,14 @@ proc repair_timing_helper { {hold_margin 1} } {
   log_cmd repair_timing {*}$additional_args
 }
 
-proc repair_design_helper {} {
+proc repair_design_helper { {extra_args ""} } {
```

Comment:
Just a heads up, used in global_route.tcl too. Should be fine near as I can see.

```
$ grep -r repair_design_helper scripts/*
scripts/floorplan.tcl:  repair_design_helper
scripts/floorplan.tcl:  repair_design_helper
scripts/global_route.tcl:    repair_design_helper
scripts/resize.tcl:repair_design_helper
scripts/util.tcl:proc repair_design_helper {} {
```